src/Gui/Quarter/QuarterWidget.cpp

```diff
@@ -289,7 +289,7 @@ QuarterWidget::QuarterWidget(QtGLContext * context, QWidget * parent, const QtGL
 void
 QuarterWidget::constructor(const QtGLFormat & format, const QtGLWidget * sharewidget)
 {
-  QGraphicsScene* scene = new QGraphicsScene;
+  QGraphicsScene* scene = new QGraphicsScene(this);
```

Comment:
https://doc.qt.io/qt-5/qgraphicsview.html#setScene
The view does not take ownership of scene.

Reply:
I Agree, it should

Reply:
17ec9b7c08708